_data/projects/axelrod.yml

```diff
@@ -1,12 +1,10 @@
-name: pretix
-desc: pretix is a conference ticket web shop that was started for MRMCD, a German non-profit IT conference, and aims to be a very flexible and configurable ticket setting solution.
-site: https://github.com/pretix/pretix
+name: Axelrod-Python
+desc: A research tool to run Iterated Prisoner's Dilemma tournaments.
+site: http://axelrod.readthedocs.io/en/latest/
 tags:
 - oss
-- web
 - python
-- django
-- javascript
+- gametheory
 upforgrabs:
   name: easy_pull_request
   link: https://github.com/Axelrod-Python/Axelrod/issues?q=is%3Aissue+is%3Aopen+label%3AEasy_pull_request
```

Comment:
While we're here, flipping this to https://github.com/Axelrod-Python/Axelrod/labels/Easy_pull_request will enable the client-side issue count check.